src/tools/validator/validate-json.ts

```diff
@@ -47,10 +67,14 @@ const validationPromises = Object.keys(schemaToJsonLookup).map<
 // up by CI.
 Promise.all(validationPromises)
   .then((validationResults) => {
-    const flatResult = validationResults.flatMap((bools) => bools);
+    const flatResult = validationResults.flatMap(
+      (booleanResults) => booleanResults
+    );
+
```

Comment:
You're not really mapping anything so I think this will do

```suggestion
    const flatResult = validationResults.flat()
```

Reply:
Aha, I wasn't familiar with flat() haha, thanks